pipelines/internal/commands/run/run.go

```diff
@@ -200,7 +206,7 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 
 		cancelOnInterruptOrTimeout(ctx, service, lro.Name, *timeout, abort)
 
-		fmt.Printf("Pipeline running as %q\n", lro.Name)
+		fmt.Printf("Pipeline running as %q (attempt: %d, preemptible: %t)\n", lro.Name, attempt, req.Pipeline.Resources.VirtualMachine.Preemptible)
```

Comment:
I think we should do attempt + 1 as I'd said in the previous comment: otherwise this says attempt: 0 on the first run.

Reply:
I initialized attempt with 1. I tested it and it starts correctly